internal/model/rwfolder.go

```diff
@@ -1228,12 +1227,14 @@ loop:
 
 			if len(batch) == maxBatchSize {
 				p.model.updateLocals(p.folder, batch)
+				p.model.receivedFile(p.folder, batch[len(batch)-1].Name)
```

Comment:
Can't move it into updateLocals because that's also used for updates from the scanner.